cmd/di.go

```diff
@@ -329,10 +331,22 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.OptionsNetwork)
 		network.EtherClientRPC = options.EtherClientRPC
 	}
 
+	if options.StatisticsAddress != metadata.DefaultNetwork.ELKAddress {
+		network.ELKAddress = options.StatisticsAddress
+	}
+
 	di.NetworkDefinition = network
 	di.MysteriumAPI = mysterium.NewClient(network.DiscoveryAPIAddress)
 	di.MysteriumMorqaClient = oracle.NewMorqaClient(network.QualityOracle)
 
+	var eventsTransport events.Transport
+	if options.DisableStatistics {
```

Comment:
I'd rather this be an opt-in instead of an opt-out since we're working on privacy. But that's just me.

Reply:
You'll also want to make sure that we disable these on the e2e suite if we stay with the opt-out.

Reply:
We discussed this with @zolia that we probably won't receive any stats if it is an opt-in. Do you think we all need to discuss more about this topic?

Reply:
Discussed this once more time all together, decided to keep it opt-out.